modules/core/server/views/partials/footer.server.view.html

```diff
@@ -2,31 +2,47 @@
         role="contentinfo"
         class="container hidden-print hidden-xs"
         ng-if="!app.isFooterHidden">
-  <div class="row">
-    <div class="col-xs-12" itemscope itemtype="http://schema.org/Organization">
-      <a ui-sref="foundation" itemprop="name">
-        Trustroots Foundation
-      </a>
-      <link itemprop="url" href="https://www.trustroots.org/">
-      <link itemprop="logo" href="https://www.trustroots.org/img/logo/color.png">
-
-      <ul class="list-inline text-center">
+  <hr>
+  <div class="navbar">
+    <div class="col-xs-6">
+      <ul class="navbar">
```

Comment:
I don't think `navbar` has any effect here?

The list would work nicely without bulletins:

<img width="197" alt="image" src="https://user-images.githubusercontent.com/87168/49526860-6318ad80-f8b9-11e8-9840-382f8ee35a4f.png">

Using `list-unstyled` does the trick (https://getbootstrap.com/docs/3.3/css/#unstyled):

<img width="208" alt="image" src="https://user-images.githubusercontent.com/87168/49527158-f7831000-f8b9-11e8-92ba-6969ffdfed36.png">